docs/reference/glossary.md

```diff
@@ -19,7 +19,7 @@ aufs (advanced multi layered unification filesystem) is a Linux [filesystem](#fi
 Docker supports as a storage backend. It implements the
 [union mount](http://en.wikipedia.org/wiki/Union_mount) for Linux file systems.
 
-## Base image
+## base image
 
 An image that has no parent is a **base image**.
```

Comment:
Not related to your PR, but wondering if we should expand this definition, because "base image" is also often used to indicate the "parent" image for user's images.

For example, in the following Dockerfile;

```Dockerfile
FROM mysql:5.7
COPY foo /bar
```

I think `mysql:5.7` is commonly called the "base image for your Dockerfile", but the official `mysql` image is in itself based on `debian:jessie`, so wouldn't match the current definition.